.huskyrc.js

```diff
@@ -0,0 +1,16 @@
+const tasks = arr => arr.join(' && ')
+module.exports = {
+  'hooks': {
+    'pre-commit': tasks([
+      'exec < /dev/tty',
+      `if ! [ -x "$(command -v circleci2)" ]; then
```

Comment:
why is this looking for the `circleci2` command, but then using the `circleci` command?

Reply:
oops, was making sure it worked properly for invalid commands. thanks for catching that